hypex/comparators/abstract.py

```diff
@@ -125,41 +138,168 @@ def _extract_dataset(
         return Dataset.from_dict(compare_result, roles, BackendsEnum.pandas)
 
     # TODO выделить в отдельную функцию с кваргами (нужно для альфы)
+
+    @staticmethod
+    def _split_ds_into_columns(
+        data: List[Tuple[str, Dataset]]
+    ) -> List[Tuple[str, Dataset]]:
+        result = []
+        for bucket in data:
```

Comment:
may be it is possible to use generator here

Reply:
```a = [(f"{bucket[0]}{NAME_BORDER_SYMBOL}{column}", bucket[1][column]) for bucket in data for column in bucket[1].columns]``` 
maybe like this

Reply:
done